.golangci.yml

```diff
@@ -11,7 +11,9 @@ issues:
   # Set to 0 to disable.
   # Default: 3
   max-same-issues: 0
-  new-from-rev: ""
+  # Exclude previously existing issues from the wrapcheck report
```

Comment:
nit: this comment should just say "Exclude previously existing issues from the report", as it applies to all reports (although in this case only the `wrapcheck` has violations)

Reply:
Took care of it. Thanks!